From 720a2c185e792b289ce52a56aa15857faa8fa35a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 22 Apr 2018 16:17:23 +0200
Subject: [PATCH] abuild.in: don't fail if git describe fails

`git describe` by default looks for tags, but `git clone` does not clone
tags by default which causes failures on travis currently.

Also redirect `git describe` errors to /dev/null while being here.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 67a596f..2c6feb4 100644
--- a/abuild.in
+++ b/abuild.in
@@ -1548,7 +1548,7 @@ update_abuildrepo_index() {
 	subpkg_unset
 
 	if [ -z "$DESCRIPTION" ]; then
-		DESCRIPTION="$repo $(cd $startdir && $git describe)"
+		DESCRIPTION="$repo $(cd $startdir && $git describe 2>/dev/null || true)"
 	fi
 
 	for i in $allarch; do
-- 
2.17.0

